lib/fastlane/plugin/ionic/actions/ionic_action.rb

```diff
@@ -101,6 +102,8 @@ def self.build(params)
         args << '--prod' if params[:prod]
         args << '--browserify' if params[:browserify]
 
+        ionic_path = params[:use_local_ionic] ? "node_modules/.bin/" : ""
```

Comment:
You can make `ionic_path` a method to avoid the repetition here 🚀

Reply:
Done!